src/components/swap/index.tsx

```diff
@@ -22,6 +22,7 @@ import { colors } from '@/lib/styles/colors'
 import { isValidTokenInput } from '@/lib/utils'
 import { getNativeToken, getTokenBySymbol } from '@/lib/utils/tokens'
 
+import { useAnalytics } from '@/lib/hooks/use-analytics'
```

Comment:
minor: this should actually throw lint/sort warning. import should be in the above group.